src/components/post/post-container.css

```diff
@@ -0,0 +1,44 @@
+.post_container {
+  width: 70vw;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 6rem;
+}
+.ant-space-item .post_image_wrapper {
+  width: fit-content;
+  margin-left: auto;
+  margin-right: auto;
+}
+.post_image_wrapper_outer {
+  background-color: #f5f5f4;
+}
+.add_comment_avatar {
+  margin-right: 10px;
+}
+.comment_row {
+  margin-top: 8px;
+}
+.add_comment_button {
+  margin-left: 8px;
+}
+
+@media only screen and (max-width: 1600px) {
```

Comment:
nice use of media queries, it encloses almost every screen size

Reply:
Thanks